UnitTesting/Constants.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Reflection;
+
+namespace UnitTesting
+{
+    public static class Constants
+    {
+        public static string connection_string = "Server=localhost;Database=DotNetCourseDatabase;TrustServerCertificate=true;Trusted_Connection=true;";
+        private static Random random = new Random();
+
+        public static decimal GetRandomDecimal(decimal minValue, decimal maxValue)
+        {
+            double randomDouble = random.NextDouble();
+            decimal scaledValue = (decimal)randomDouble * (maxValue - minValue) + minValue;
+
+            return scaledValue;
+        }
+
+        public static bool CompareFields<T>(T obj1, T obj2)
```

Comment:
no need for this helper method. you can use FluentAssertions for that:

`bet.Should().BeEquivalentTo(otherBet)`